metrics/github/repos.py

```diff
@@ -1,4 +1,17 @@
-NON_TECH_REPOS = {
+from metrics.github import query
+
+
+def our_repos(client, org):
```

Comment:
would "tech" possibly be better than "our" here? Who we are?

Reply:
Yeah, I'll buy that. Thanks.